app/views/users/backup_code_setup/confirm_backup_codes.erb

```diff
@@ -0,0 +1,31 @@
+<% title t('titles.backup_codes') %>
+
+<%= render PageHeadingComponent.new.with_content(t('titles.backup_codes')) %>
+
+<p>
+  <%= t('two_factor_authentication.backup_codes.warning_html') %>
+</p>
+
+<p>
+  <%= t('two_factor_authentication.backup_codes.instructions', app_name: APP_NAME) %>
+</p>
+
+<div>
+  <%= link_to(
+        sign_up_completed_path,
+        class: 'usa-button usa-button--wide usa-button--big margin-bottom-3 margin-top-5',
+      ) do %>
+    <%= t('two_factor_authentication.backup_codes.saved_backup_codes') %>
+  <% end %>
+
+  <%= link_to(
+        backup_code_regenerate_path,
+        class: 'usa-button usa-button--wide usa-button--big usa-button--outline',
+      ) do %>
+    <%= t('two_factor_authentication.backup_codes.new_backup_codes') %>
+  <% end %>
+</div>
```

Comment:
Couple minor comments:

- Do we need the empty `<div>` wrapper?
- `ButtonComponent` can help abstract the specifics of the classes, though it is a little clunky to use for links like this (I'd like to see the component enhanced with something like a `url` option for this common use-case)
- Might want to get some clarity on design with these, since...
   - ...the margins are correct per the design, but non-standard with how other pages space the primary buttons (other pages only have 2.5rem above primary button, this one is 3.5rem)
   - ...the buttons in the design have a constrained width, but that's also not really standard in the design system. If needed, we could use [grid utilities](https://designsystem.digital.gov/utilities/layout-grid/) to constrain the width to 50% at larger viewports?

```suggestion
<%= render ButtonComponent.new(
      action: ->(**tag_options, &block) { link_to(sign_up_completed_path, **tag_options, &block) },
      big: true,
      wide: true,
      class: 'margin-y-3'
    ).with_content(t('two_factor_authentication.backup_codes.saved_backup_codes')) %>

<%= render ButtonComponent.new(
      action: ->(**tag_options, &block) { link_to(backup_code_regenerate_path, **tag_options, &block) },
      big: true,
      wide: true,
      outline: true,
    ).with_content(t('two_factor_authentication.backup_codes.new_backup_codes')) %>
```

Reply:
We will need the `<div>`, but it will be to set up the grid for the two buttons.

For the margin, I'll get it to `2.5rem`. As for the buttons, since the buttons are constrained on desktop, I'll apply it on all viewports